base.py

```diff
@@ -222,14 +233,15 @@ def print_exceptions(self, detailed):
                     rule_suffix = "s"
                 print "{0:10d} {1} {2} message{3}".format(
                     rule_count, rule_class_name, e_type_name, rule_suffix)
-                if detailed:
-                    for exception in self.exceptions[e_type][rule_class]:
-                        if exception.error_log:
-                            for error in exception.error_log:
-                                print " "*14+"Line {0}: {1}".format(
-                                    error.line, error.message)
-                        else:
-                            print " "*14+"{0}".format(exception)
+                if log_level > 0:
+                    if e_type in exception_types[0:log_level]:
```

Comment:
That is so that a summary of other log levels are that have not been selected is shown. For example with -v, it would show detailed messages for errors but only show the summary for Warning and Info messages.